style.css

```diff
@@ -0,0 +1,104 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1c1a19;
+}
+
+a {
+  text-decoration: none;
+}
+  
+li {
+  list-style: none;
+  flex-direction: row;
+}
+ 
+.container {
+  padding-left: 24px;
+  padding-right: 24px;
+}
+  
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: white;
+}
+  
+header a {
+  color: white;
+  text-decoration: none;
+  font-size: 17px;
+}
+  
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 48px;
+  justify-content: space-between;
+  font-family: 'Roboto', sans-serif;
+  background: #3c3a39;
+  align-items: center;
+}
+
+.hero-wrapper {
+  /* position:absolute; */
+  background-image: url(./assets/illustration-header-mobile.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-position: 100% 70%;
+  background-attachment: fixed;
+  height: 100vh;
+}
```

Comment:
- [ ] Great job making some of the changes required by the previous reviewer. So, I noticed that you used `flexbox` on each content of the `headline section` instead of using it(flexbox) on the **container**. i.e. you are supposed to use the **flexbox** he pointed out at this **section container** to align all contents inside the section. I know you can do it :muscle: :muscle: